test/factory/ZoraCreator1155Factory_Fork.t.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "forge-std/Test.sol";
+import {IZoraCreator1155Factory} from "../../src/interfaces/IZoraCreator1155Factory.sol";
+import {IZoraCreator1155} from "../../src/interfaces/IZoraCreator1155.sol";
+import {IMinter1155} from "../../src/interfaces/IMinter1155.sol";
+import {IOwnable} from "../../src/interfaces/IOwnable.sol";
+import {ICreatorRoyaltiesControl} from "../../src/interfaces/ICreatorRoyaltiesControl.sol";
+import {MockContractMetadata} from "../mock/MockContractMetadata.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {ZoraCreatorFixedPriceSaleStrategy} from "../../src/minters/fixed-price/ZoraCreatorFixedPriceSaleStrategy.sol";
+import {MintFeeManager} from "../../src/fee/MintFeeManager.sol";
+
+import {ChainConfig, Deployment} from "../../script/ZoraDeployerBase.sol";
+
+abstract contract ForkTesterBase is Test {
```

Comment:
a copy in a way of `ZoraDeployerBase` - but chainId comes from block.chainId, and inherits from `Test`